src/QueryDataTable.php

```diff
@@ -735,7 +735,7 @@ protected function showDebugger(array $output): array
         $query_log = $this->getConnection()->getQueryLog();
         array_walk_recursive($query_log, function (&$item) {
             if (is_string($item)) {
-                $item = utf8_encode($item);
+                $item = iconv('iso-8859-1', 'utf-8', $item);
```

Comment:
Sorry, not familiar much with ISO. Is this needed/part of the deprecation too? If not, maybe we can separate this into another PR.

Reply:
`utf8_encode` and `utf8_decode` get deprecated in `8.2`. See [here](https://www.php.net/manual/en/function.utf8-encode.php)

The `iconv` call is equivalent to the `utf8_encode` call as per the PHP manual page linked above.

Reply:
thanks for the info, learn something new :)